services/src/main/java/com/jos/dem/jmailer/service/MailService.java

```diff
@@ -20,5 +20,6 @@
 
 public interface MailService {
 
-  void sendMailWithTemplate(Map<String, String> values, Map model, String template);
+  void
+  sendMailWithTemplate(Map<String, String> values, Map model, String template);
```

Comment:
I think that change is not necessary

Reply:
I forgot to format the code; I fixed it.